cmd/kubectl-gadget/top/file.go

```diff
@@ -15,62 +15,50 @@
 package top
 
 import (
-	"encoding/json"
 	"fmt"
-	"os"
 	"strconv"
 	"strings"
-	"sync"
-	"time"
 
 	"github.com/spf13/cobra"
-	"golang.org/x/term"
 
 	commonutils "github.com/inspektor-gadget/inspektor-gadget/cmd/common/utils"
 	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
-	gadgetv1alpha1 "github.com/inspektor-gadget/inspektor-gadget/pkg/apis/gadget/v1alpha1"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/top"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/top/file/types"
 )
 
-type FileFlags struct {
-	CommonTopFlags
-
-	ParsedSortBy types.SortBy
-	AllFiles     bool
-}
-
 type FileParser struct {
 	commonutils.BaseParser[types.Stats]
-	sync.Mutex
 
-	flags     *FileFlags
-	nodeStats map[string][]types.Stats
+	flags *CommonTopFlags
 }
 
 func newFileCmd() *cobra.Command {
-	var flags FileFlags
-
-	commonFlags := &utils.CommonFlags{
-		OutputConfig: commonutils.OutputConfig{
-			// The columns that will be used in case the user does not specify
-			// which specific columns they want to print.
-			CustomColumns: []string{
-				"node",
-				"namespace",
-				"pod",
-				"container",
-				"pid",
-				"comm",
-				"reads",
-				"writes",
-				"r_kb",
-				"w_kb",
-				"t",
-				"file",
+	commonTopFlags := &CommonTopFlags{
+		CommonFlags: utils.CommonFlags{
+			OutputConfig: commonutils.OutputConfig{
+				// The columns that will be used in case the user does not specify
+				// which specific columns they want to print.
+				CustomColumns: []string{
+					"node",
+					"namespace",
+					"pod",
+					"container",
+					"pid",
+					"comm",
+					"reads",
+					"writes",
+					"r_kb",
+					"w_kb",
+					"t",
+					"file",
+				},
 			},
 		},
 	}
 
+	var AllFiles bool
```

Comment:
I think it's good to have a `FileFlags` struct as it was before. It would help to generalize it once we want to support it on local-gadget. (I'm find with removing the struct now and adding it back when needed)